chargedUp/src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
This file is cracked really good foundation for the arm subsystem

Reply:
:D